invoices/update.go

```diff
@@ -49,6 +49,10 @@ const (
 	// ResultSettled is returned when we settle an invoice.
 	ResultSettled
 
+	// ResultCancelled is returned when we cancel an invoice and its associated
+	// htlcs.
+	ResultCancelled
```

Comment:
We did a recent full repo rename to standardize on `Canceled`

Reply:
This spelling hurts me but ok :(